apps/ledger-live-desktop/src/renderer/modals/Send/steps/StepConfirmation.tsx

```diff
@@ -75,11 +77,14 @@ function StepConfirmation({
           currencyName={currencyName}
         />
         {signed ? (
-          <BroadcastErrorDisclaimer
-            title={<Trans i18nKey="send.steps.confirmation.broadcastError" />}
+          <NodeError
+            error={error}
+            currencyName={String(currencyName)}
+            networkName={String(mainAccount?.currency.name)}
```

Comment:
In my understanding here `currencyName` and `mainAccount?.currency.name` are the same thing.
<img width="405" alt="image" src="https://github.com/user-attachments/assets/ecaa6b4a-7606-439f-b2eb-30b26ba82cf6">

In LLM I used `currency.familly` for the network (eg. evm) and `currency.ticker` for the coin:
E.g:
<img width="405" alt="image" src="https://github.com/user-attachments/assets/a5c8da1f-8720-4d82-b69c-5380a0480fbc">

WDYT ?

Reply:
hmmm not sure for the curency.familly evm is not a network it's based on Eth network. So for example we need to display on the Base/Scroll/... network

I've added the ticker too 
<img width="1624" alt="image" src="https://github.com/user-attachments/assets/96e4f583-2061-49e0-9829-d9cbc98afe3f">